README

```diff
@@ -12,29 +16,37 @@ they are each experiements, some with mature development and others with minimal
 development.
 
 <=== Sub App Descriptions ===>
-    - ctimer.pl: 
-        This is a perl script that captures the pomodoro style time tracking. 
+    - ctimer.pl:
+        This is a perl script that captures the pomodoro style time tracking.
     - timetracker:
         This application is focused on tracking time as it relates to
         roles/projects/tasks. It uses YAML input/output to file in order to
         store the data.
 
-        - Notes on install, while in dev:
```

Comment:
There are several changes here that look only like formatting changes. Are the necessary?